src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <!-- Mobile and legacy targets - comment these out or set env variable INCLUDE_MOBILE_AND_LEGACY_TFM to speedup the build -->
-  <PropertyGroup Condition="'$(INCLUDE_MOBILE_AND_LEGACY_TFM)' != ''">
+  <PropertyGroup Condition="'$(INCLUDE_MOBILE_AND_LEGACY_TFM)' == ''">
```

Comment:
We discussed not including these targets by default. This is to simplify the lives of external contributors.

Reply:
reverted the change and adding `INCLUDE_MOBILE_AND_LEGACY_TFM` to the build pipelines. running a test build now